pbf/views.py

```diff
@@ -249,6 +249,12 @@ def new_game(request):
     'LocusSerpent': ['Elemental Aegis'],
     }
 
+def render_player_view(request, player):
+    if player.spirit.name == 'Earthquakes':
+        impending_with_energy = GamePlayerImpendingWithEnergy.objects.filter(gameplayer=player)
```

Comment:
need to set `impending_with_energy` in the case where the spirit is any spirit other than Dances Up Earthquakes, otherwise the line below `impending_with_energy` references undefined variable

Reply:
Thanks!